FirebaseAdmin/FirebaseAdmin.Tests/Messaging/MessageTest.cs

```diff
@@ -75,6 +75,10 @@ public void Notification()
                     Title = "title",
                     Body = "body",
                 },
+                FcmOptions = new FcmOptions()
```

Comment:
Can you also add a test case to see what happens when the AnalyticsLabel is unspecified (i.e. null)? I think the current implementation will fail in that case.